projection/javadsl/src/main/java/com/lightbend/lagom/javadsl/projection/Projections.java

```diff
@@ -9,18 +9,59 @@
 import akka.annotation.ApiMayChange;
 import com.lightbend.lagom.projection.State;
 
-// https://github.com/lagom/lagom/issues/2048
-/** TODO: docs */
+/**
+ * Public interface to query the internal projection registry that keeps track of the {@link State}
+ * of each projection. A projections is a process consuming an Akka Persistence Journal handling
+ * each event into a read table of a broker topic. In Lagom the erm projection only refers to <code>
+ * ReadSideProcessor</code>'s and <code>TopicProducers</code>'s (not Broker subscribers). This is
+ * different than the meaning of projection in akka/akka-projection.
```

Comment:
akka/akka-projection is not a thing yet. should we refer to it here?

Reply:
I almost deleted that sentence when copying this from the design docs, but then I remembered https://github.com/akka/akka-projection already exists so I decided to just leave the soft reference...